data-processed/Imperial-ensemble1/2020-05-03-Imperial-ensemble1.csv

```diff
@@ -0,0 +1,385 @@
+forecast_date,target,target_end_date,location,type,quantile,value
+2020-05-03,1 day ahead cum death,2020-05-04,US,point,NA,69636
+2020-05-03,1 day ahead inc death,2020-05-04,US,point,NA,1954
+2020-05-03,1 wk ahead cum death,2020-05-09,US,point,NA,79511
+2020-05-03,1 wk ahead inc death,2020-05-09,US,point,NA,1985
```

Comment:
@sangeetabhatia03 The issue I see is that on this line the "1 wk ahead inc death" looks like it is just one day's incident deaths count, not the sum of all seven days from that week. Does that make sense?

Reply:
I.e. this line has the identical `value` as line 15 with the "6 day ahead inc death".